lib/jekyll-mentions.rb

```diff
@@ -2,28 +2,36 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Mentions < Jekyll::Generator
+  class Mentions < Jekyll::Converter
+    MENTIONABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
```

Comment:
The extension given doesn't change. The priority can be used to ensure it's the last converter but that's all. Maybe we can restructure this in Jekyll?